app/views/govuk_publishing_components/components/_contextual_breadcrumbs.html.erb

```diff
@@ -0,0 +1,20 @@
+<% navigation = GovukPublishingComponents::Presenters::ContextualNavigation.new(content_item, request.path) %>
+
+<% if navigation.step_nav_helper.show_header? %>
+  <!-- Rendering step by step nav -->
+  <%= render 'govuk_publishing_components/components/step_by_step_nav_header', navigation.step_nav_helper.header %>
+<% else %>
+  <% if navigation.should_present_taxonomy_navigation? %>
+    <!-- Rendering taxonomy breadcrumbs -->
+    <% if navigation.taxon_breadcrumbs.any? %>
```

Comment:
Should this condition be folded into the `should_present_taxonomy_navigation?` method? Or [is it superfluous now](https://github.com/alphagov/govuk_publishing_components/pull/228/files#diff-ad589f3ed0c6ff0d7aa6037b73e96bedR14)? I realise you've ported much of this functionality but would we ever deliberately want to show nothing here?

Reply:
Yes, I think so. I'll do that in a follow up PR!